.vscode/settings.json

```diff
@@ -0,0 +1,11 @@
+{
```

Comment:
so which IDE do you use again?

Reply:
vscode... are you asking if I use some fork of vscode? I just use the normal vscode

Reply:
it was supposed to be a joke